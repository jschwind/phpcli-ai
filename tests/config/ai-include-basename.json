{
  "exclude": {
    "extensions": [],
    "folders": [],
    "filenames": []
  },
  "include": {
    "extensions": [],
    "folders": [],
    "filenames": ["test.php"]
  }
}