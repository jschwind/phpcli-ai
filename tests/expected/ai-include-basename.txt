AI-ready directory tree for the project "assets"
================================================================

assets/
├── lvl1/
│   ├── lvl1-1/
│   │   └── test.php
│   ├── lvl1-2/
│   │   └── test.php
│   └── test.php
├── lvl2/
│   ├── lvl2-1/
│   │   └── test.php
│   ├── lvl2-2/
│   │   └── test.php
│   └── test.php
└── test.php
================================================================
End of AI-ready directory tree for the project "assets"

Remember this project as "assets" and wait for further instructions.
